src/app/account/additional-components/account-flows-tab/account-flows-subtabs/account-flows-datasets-subtab/account-flows-datasets-subtab.component.ts

```diff
@@ -118,105 +125,87 @@ export class AccountFlowsDatasetsSubtabComponent extends BaseComponent implement
 
     public readonly ProcessCardFilterMode: typeof ProcessCardFilterMode = ProcessCardFilterMode;
     public readonly DatasetViewTypeEnum: typeof DatasetViewTypeEnum = DatasetViewTypeEnum;
-    private readonly CARDS_FLOW_PROCESSES_PER_PAGE: number = 9;
 
     public ngOnInit(): void {
-        this.dashboardFiltersState.isFirstInitialization = true;
+        this.accountFlowsFiltersService.updateFilters({ isFirstInitialization: true });
         this.fetchCardsData();
     }
 
     public get dashboardFiltersState(): DashboardFiltersOptions {
         return this.accountFlowsFiltersService.currentFiltersSnapshot;
     }
 
-    public getPageFromUrl(): void {
-        const pageParam = this.activatedRoute.snapshot.queryParamMap.get(ProjectLinks.URL_QUERY_PARAM_PAGE);
-        if (pageParam) {
-            this.currentPage = +requireValue(pageParam);
-        }
-    }
-
-    public toggleWebhookCardState(params: {
-        datasetBasics: DatasetBasicsFragment;
-        subscriptionId: string;
-        state: FlowProcessEffectiveState;
-    }): void {
-        const result$: Observable<boolean> =
-            params.state === FlowProcessEffectiveState.Active
-                ? this.datasetWebhooksService.datasetWebhookPauseSubscription(
-                      params.datasetBasics.id,
-                      params.subscriptionId,
-                  )
-                : this.datasetWebhooksService.datasetWebhookResumeSubscription(
-                      params.datasetBasics.id,
-                      params.subscriptionId,
-                  );
-        result$.pipe(take(1)).subscribe((result: boolean) => {
-            if (result) {
-                setTimeout(() => {
-                    this.refreshNow();
-                }, this.TIMEOUT_REFRESH_FLOW);
-            }
-        });
-    }
-
     public fetchCardsData(): void {
-        this.accountFlowsCardsData$ = timer(0, environment.delay_polling_ms).pipe(
-            tap(() => {
-                this.getPageFromUrl();
-            }),
+        const polling$ = timer(0, environment.delay_polling_ms);
+        const triggerWithInitial$ = this.fetchTrigger$.pipe(startWith(undefined));
+        this.accountFlowsCardsData$ = combineLatest([polling$, triggerWithInitial$]).pipe(
             switchMap(() =>
                 this.accountService.getAccountFlowsAsCards({
                     accountName: this.accountName,
                     page: this.currentPage - 1,
-                    perPage: this.CARDS_FLOW_PROCESSES_PER_PAGE,
+                    perPage: this.processesPerPage,
```

Comment:
So, with this algorithm, if we loaded 3 pages, then clicked pause on the last item, we get a refresh from page 1 with 3x size. And then, if i scroll more, I would load 3x next page, instead of normal. Not saying this is clearly wrong, but very suspicious.